bin/cwutil

```diff
@@ -52,11 +52,11 @@ def stats(namespace, metric_name, dimensions=None, statistics="Average", start_t
     if end_time:
         end_time = _parse_time(end_time)
     else:
-        end_time = datetime.utcnow()
+        end_time = datetime.now(tz=timezone.utc).replace(tzinfo=None)
```

Comment:
`datetime.now(tz=timezone.utc)` provides `tzinfo` as extra information than `datetime.utcnow()`

example:
```
>>>datetime.now(tz=datetime.timezone.utc)
datetime.datetime(2024, 11, 25, 16, 27, 1, 314939, tzinfo=datetime.timezone.utc)
>>> datetime.utcnow()
<stdin>:1: DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
datetime.datetime(2024, 11, 25, 16, 27, 18, 347221)
>>> 
```

So to make sure the results are same as previous, we have to replace `tzinfo` with `None` so that the extra attribute `tzinfo` will not cause any issue.